portal/src/main/webapp/WEB-INF/jsp/global/footer.jsp

```diff
@@ -34,7 +34,7 @@
 
 <div id="footer">
     <a href="http://cbioportal.org">cBioPortal</a>
-    <span id="footer-span-version"> Version ${project.version}</span>
+    <span id="footer-span-version"> Version 1.7.0-SNAPSHOT</span>
```

Comment:
@alisman oops that's not on purpose, thanks! Copied over the one from target folder. This apparently is overwritten in build